Antidote/CallsManager.m

```diff
@@ -84,14 +88,18 @@ - (void)dealloc
 
 - (void)callToChat:(OCTChat *)chat
 {
+    AALogVerbose(@"%@", chat);
+
     if (self.currentCall) {
         NSAssert(NO, @"We should not be able to make another call yet");
         return;
     }
 
-    OCTCall *call = [self.manager callToChat:chat enableAudio:YES enableVideo:NO error:nil];
+    NSError *error;
+    OCTCall *call = [self.manager callToChat:chat enableAudio:YES enableVideo:NO error:&error];
 
     if (! call) {
+        AALogWarn(@"error %@, failureReason:%@", error, error.localizedFailureReason);
```

Comment:
Isn't failure reason included in error description, that you pass as first argument? I think it should be there as well as domain and error code.